### PR TITLE
config: change the requested reviewer for docs-dm repo

### PR DESCRIPTION
**Pull request by qiancai**

### CODE REVIEW

**Comment by Mini256** [comment]
I want to confirm whether the bot will now request reviews from these newly added excluded reviewers.

In fact, `include_reviewers` and `exclude_reviews` options cannot be configured at the same time, after the repository is configured with `include_reviewers`, the robot will only request reviewers in the list.

**Review by Liuxiaozhen12 — APPROVED** [review]

**Comment by Liuxiaozhen12** [comment]
> I want to confirm whether the bot requests reviews from these newly added excluded reviewers now.
> 
> In fact, `include_reviewers` and `exclude_reviews` options cannot be configured at the same time, after the repository is configured with `include_reviewers`, the bot will only request reviewers in the list.

Right now, the bot request reviews from CharLotteiu who is in the newly added excluded reviewers. We mainly want to change  the reviewer of docs-dm repository.

File: prow/config/external_plugins_config.yaml [hunk 1]
```diff
@@ -1032,24 +1032,40 @@ ti-community-blunderbuss:
     max_request_count: 1
     include_reviewers:
       - TomShawn
+    exclude_reviewers:
+      # Inactive committers
+      - CharLotteiu  
+      -  yikeke
```
Comment (Liuxiaozhen12) [inline prow/config/external_plugins_config.yaml:1038]:
```suggestion
```

File: prow/config/external_plugins_config.yaml [hunk 2]
```diff
@@ -1032,24 +1032,40 @@ ti-community-blunderbuss:
...
   - repos:
       - pingcap/docs-cn
     pull_owners_endpoint: https://prow.tidb.io/ti-community-owners
     max_request_count: 1
     include_reviewers:
       - TomShawn
+    exclude_reviewers:
+      # Inactive committers
+      - CharLotteiu  
+      -  yikeke
```
Comment (Liuxiaozhen12) [inline prow/config/external_plugins_config.yaml:1048]:
```suggestion
```

File: prow/config/external_plugins_config.yaml [hunk 3]
```diff
@@ -1032,24 +1032,40 @@ ti-community-blunderbuss:
...
       - pingcap/docs-dm
     pull_owners_endpoint: https://prow.tidb.io/ti-community-owners
     max_request_count: 1
     include_reviewers:
-      - CharLotteiu
+      - Liuxiaozhen12
+    exclude_reviewers:
+      # Inactive committers
+      - CharLotteiu  
+      -  yikeke
```
Comment (Liuxiaozhen12) [inline prow/config/external_plugins_config.yaml:1058]:
```suggestion
```

File: prow/config/external_plugins_config.yaml [hunk 4]
```diff
@@ -1032,24 +1032,40 @@ ti-community-blunderbuss:
...
   - repos:
       - pingcap/docs-tidb-operator
     pull_owners_endpoint: https://prow.tidb.io/ti-community-owners
     max_request_count: 1
     include_reviewers:
       - qiancai
+     exclude_reviewers:
+      # Inactive committers
+      - CharLotteiu  
+      -  yikeke
```
Comment (Liuxiaozhen12) [inline prow/config/external_plugins_config.yaml:1068]:
```suggestion
```

**Review by Liuxiaozhen12 — COMMENTED** [review]
PTAL, thanks!

**Review by Mini256 — APPROVED** [review]

**Comment by qiancai** [comment]
Thanks @Mini256 for the review and @Liuxiaozhen12 for the update.   @Mini256 If there is no other issue, would you please merge this PR? Thanks!

**Pull request merged**